core/src/main/scala/org/apache/spark/scheduler/LiveListenerBus.scala

```diff
@@ -54,12 +54,6 @@ private[spark] class LiveListenerBus(conf: SparkConf) {
   // Indicate if `stop()` is called
   private val stopped = new AtomicBoolean(false)
 
-  /** A counter for dropped events. It will be reset every time we log it. */
-  private val droppedEventsCounter = new AtomicLong(0L)
-
-  /** When `droppedEventsCounter` was logged last time in milliseconds. */
-  @volatile private var lastReportTimestamp = 0L
-
```

Comment:
@wangyum, I think we don't usually open a PR to fix a couple of nits. Such changes could be made together when somebody touches here.

Reply:
+1 for @HyukjinKwon 's comment.